### ISSUE TITLE
Loss computation prompt tuning

### ISSUE THREAD

**Issue by apiro** [huggingface/peft#143]
I think there is an error in the loss computation. See screen below, the denominator is inverted ! If the size of the sets is the same (your case) it's ok but if you want to put more data in the training set the losses will be wrong :) 

![Screenshot 2023-02-28 at 18 16 12](https://user-images.githubusercontent.com/2962472/221946576-d9693739-b728-4a04-a926-60dc2c33e358.png)
